static/app/views/alerts/rules/metric/ruleForm.tsx

```diff
@@ -922,18 +922,24 @@ class RuleFormContainer extends DeprecatedAsyncComponent<Props, State> {
       value === AlertRuleComparisonType.CHANGE
         ? this.state.comparisonDelta ?? DEFAULT_CHANGE_COMP_DELTA
         : undefined;
-    const timeWindow = this.state.comparisonDelta
```

Comment:
What are these changes doing?

Reply:
We need to set a default for the time window if the user selects an anomaly detection alert

Reply:
Will there still be a time window for ones with a comparison delta? Forgive my backend eyes

Reply:
Yes, they will both be 60 minutes (just changed the condition to be more specific to detection type)

Reply:
I wonder if https://github.com/getsentry/sentry/pull/78493#discussion_r1785262190 wouldn't help address this? i think it might help with the general readability of this method.